.github/workflows/publish-assets.yaml

```diff
@@ -0,0 +1,74 @@
+name: Upload assets to releases
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Build updateCli
+      run: |
+        touch updateCli.env
+        make build.all
```

Comment:
Does the ubuntu-latest environment has golang installed?

Reply:
Apparently yes - [doc](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#go)